ios/Handlers/RNFlingHandler.m

```diff
@@ -0,0 +1,58 @@
+//
+//  RNFlingHandler.m
+//  RNGestureHandler
+//
+//  Created by Michał Osadnik on 10/04/2018.
+//  Copyright © 2018 Software Mansion. All rights reserved.
+//
+
+
+#import "RNFlingHandler.h"
+
+@implementation RNFlingGestureHandler
+
+- (instancetype)initWithTag:(NSNumber *)tag
+{
+    if ((self = [super initWithTag:tag])) {
+        _recognizer = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(handleGesture:)];
+        
+    }
+    return self;
+}
+
+- (void)configure:(NSDictionary *)config
+{
+    [super configure:config];
+    UISwipeGestureRecognizer *recognizer = (UISwipeGestureRecognizer *)_recognizer;
+
+    
+    id prop = config[@"direction"];
+    if (prop != nil) {
+        
+        NSInteger direction = [RCTConvert NSInteger:prop];
```

Comment:
Would be nice to use RCTConvert category for `UISwipeGestureRecognizerDirection` and put that logic in there, see something similar being done for example here:
https://github.com/wix/react-native-notifications/blob/master/RNNotifications/RNNotifications.m#L28